src/PoolManager.sol

```diff
@@ -93,12 +93,12 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
     /// @inheritdoc IPoolManager
     int24 public constant MIN_TICK_SPACING = TickMath.MIN_TICK_SPACING;
 
-    mapping(PoolId id => Pool.State) internal pools;
+    mapping(PoolId id => Pool.State) internal _pools;
 
     constructor(uint256 controllerGasLimit) ProtocolFees(controllerGasLimit) {}
 
     function _getPool(PoolId id) internal view override returns (Pool.State storage) {
```

Comment:
why do we have this function 😂 can you remove it? i feel like it should just be `_pools[id]` everywhere?

Reply:
it's because when we moved all the protocol fees logic into ProtocolFees.sol, it needed to have `_getPool(id)` to set the protocol fee on the pool. And since PoolManager inherits from ProtocolFees, it needs to implement that function

Reply:
ohhhhh nice, thanks for reminding me